project.yml

```diff
@@ -104,6 +104,12 @@ map:
     drop_readid: True
     drop_seq: True
 
+    # The minimal MAPQ score in order
+    # to consider a read as uniquely mapped.
+    # 1 implies no restrictions on read quality, while
```

Comment:
Alignment, not read quality.

We should document that MAPQ of 0 is assigned to multimappers (best and next-best alignment scores are equal) and that bwa mem's maximum MAPQ value is 60.
https://bioinformatics.stackexchange.com/a/2469